tests/test_reqctx.py

```diff
@@ -162,34 +166,37 @@ def g():
                 assert flask.request.args['foo'] == 'bar'
             assert not flask.request
             return 42
+
         greenlets.append(greenlet(g))
         return 'Hello World!'
 
-    rv = app.test_client().get('/?foo=bar')
+    rv = client.get('/?foo=bar')
     assert rv.data == b'Hello World!'
 
     result = greenlets[0].run()
     assert result == 42
 
+
 @pytest.mark.skipif(greenlet is None, reason='greenlet not installed')
```

Comment:
If this and the previous test were organized into a `class` specifically for tests about greenlets, then pytest supports conditionally skipping the entire class if `greenlet`s are missing rather than having to mark each test separately.